docs/getting_started/intellij_plugin.md

```diff
@@ -0,0 +1,96 @@
+This plugin provides support for working with Chutney scenarios files in IntelliJ IDEA based IDEs.
```

Comment:
I find all the images too large. Is it possible to control their sizes ?

Reply:
Which size should we use? Size can be defined using width an height in pixel